tests/features/data-visibility-and-review.feature

```diff
@@ -1,122 +1,210 @@
+  # PUBLIC ACCESS
+  # ---------------------------------------------------------------------------
+
+  @public_access @visibility
+  Scenario Outline: Public users can only access public data
+    Given I am a public API consumer
+    And there is <data_type> data stored with a visibility field that evaluates to either public or internal
+    When I request <data_type> data through unauthenticated endpoints
+    Then I should only see records where the visibility field is set to public
+    And records whose visibility field is set to internal should not be returned
+    And the response should include the review_status for each public record
+
+    Examples:
+      | data_type    |
+      | locations    |
+      | samples      |
+      | observations |
+
+  # might not be relevant yet in NMSampleLocations
+  @public_access @reports
+  Scenario: Public reports include review status disclaimers
+    Given there are public observations with review_status provisional and approved
+    When a public user generates a report
+    Then all returned observations should have visibility public
+    And provisional observations should include a disclaimer derived from review_status
+    And approved observations should indicate that they are fully reviewed
+
+  # might not be relevant yet in NMSampleLocations
+  @public_access @data_download
+  Scenario: Public data downloads exclude internal datasets
+    Given a public user requests a CSV export
+    And the dataset contains public and internal records with different review_status values
+    When the download is prepared
+    Then only the public records should be included
+    And the download should list the review_status for each record
+
+  # ---------------------------------------------------------------------------
+  # AUTHENTICATED STAFF ACCESS
+  # ---------------------------------------------------------------------------
+
+  @staff_access @visibility
+  Scenario Outline: Staff can access all data and its review status
+    Given I am an authenticated staff member
+    When I view <data_type> data through endpoints while authenticated
+    Then I should see visibility internal and public records
+    And I should see whether each record is provisional or approved
+
+    Examples:
+      | data_type    |
+      | things       |
+      | locations    |
+      | samples      |
+      | observations |
+
+  @staff_access @management
+  Scenario: Staff can change visibility without affecting review status
+    Given data is stored with visibility internal and review_status provisional
+    When an authenticated staff member updates the visibility to public
+    Then the data visibility should be persisted as public
+    And the review_status should remain provisional
+    And retrieving the data should show the updated visibility and original review_status
+
+  @staff_access @review_status
+  Scenario: Staff can approve data without changing visibility
```

Comment:
I think that we should specify that the staff member is authenticated like the previous scenario. I also think that we should specify that the staff member has the correct permissions to edit the review_status field